icaruscode/CRT/CRTPMTMatchingAna_module.cc

```diff
@@ -495,9 +497,11 @@ void icarus::crt::CRTPMTMatchingAna::analyze(art::Event const& e) {
       }
       geo::Point_t flash_pos = flashCentroid.middlePoint();
       t_m = t_m / nPMTsTriggering;
-      if (nPMTsTriggering < fnOpHitToTrigger) {
+      // F. Poppi: I will leave this commented in case we want to revert it to the original A. Scarpelli suggestion. 
+      // I'd rather have this as a variable to cut when doing the analysis.
+      /*if (nPMTsTriggering < fnOpHitToTrigger) {
```

Comment:
Doesn't Clang get upset that `fnOpHitToTrigger` is set but not used?
We'll see...

Reply:
If you leave this commented out, it may be worth either documenting why you should _not_ restore it as is, or change the code so that it would work even with this line.
The latter can be achieved by using temporary vectors and then move the content to the ones bound to the tree later, like filling a `std::vector<double> opHitX` first and finally  calling `append(fOpHitX, opHitX);` with
```cpp
template <typename Dest, typename Src>
void append(Dest& dest, Src const& src) { dest.insert(dest.end(), src.begin(), src.end()); }
```